### PR TITLE
Add Italian translation to video subtitles

### PR DESCRIPTION
**Pull request by lucrus73**
Here is the Italian translation for the home page subtitles.

### CODE REVIEW

File: assets/subtitles/lJ9C-it_IT.vtt [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+00:01:47.000 --> 00:02:06.000
+[rumore di click di tasti]
+
+46
+00:02:06.000 --> 00:02:09.000
+Sono anche inclusi gli equivalenti
+
+47
+00:02:09.000 --> 00:02:10.000
+di importantio programmi di i3
```
Comment (plfiorini) [inline assets/subtitles/lJ9C-it_IT.vtt:189]: importantio?

File: assets/subtitles/lJ9C-it_IT.vtt [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+00:02:31.000 --> 00:02:32.000
+che vi aiuterà ad imparare ad usarlo.
+
+60
+00:02:34.000 --> 00:02:36.000
+Ad oggi Sway supporta quasi tutte
+
+61
+00:02:36.000 --> 00:02:37.000
+le caratteristiche di i3s.
```
Comment (plfiorini) [inline assets/subtitles/lJ9C-it_IT.vtt:245]: i3s? perhaps it's just "i3"

**Comment by plfiorini** [comment]
Just read the text, didn't watch the video.
Where do I find it?

**Comment by emersion** [comment]
Here: https://swaywm.org/

File: assets/subtitles/lJ9C-it_IT.vtt (hunk 1 above)
Comment (lucrus73) [reply assets/subtitles/lJ9C-it_IT.vtt]: fixed

File: assets/subtitles/lJ9C-it_IT.vtt [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+00:01:33.000 --> 00:01:34.000
+È anche possibile portare le finestre
+
+39
+00:01:34.000 --> 00:01:36.000
+fuori dalla tiling area
+
+40
+00:01:36.000 --> 00:01:39.000
+e gestirle completamente con il mouse.
```
[the remark this answers is not in the document]
Comment (lucrus73) [reply assets/subtitles/lJ9C-it_IT.vtt:161]: I know the italian literal translation of "to manipulate" is "manipolare", but it's almost never used in italian in an abstract context with the meaning intended here. In italian "manipolare" refers to a the action of really moving or working with something concrete with the hands, not so when the context is a virtual cursor that moves a virtual window.
In italian "manipolare" in an abstract context always carries a negative meaning of deceptively changing things for you own personal gain.

[the remark this answers is not in the document]
Comment (masozzi) [reply assets/subtitles/lJ9C-it_IT.vtt:161]: > not so when the context is a virtual cursor that moves a virtual window.

There are no words to define that in italian.
For me "gestirle" is ok too. Mine was just a word-for-word translation.

**Comment by masozzi** [comment]
For me this can be merged.
If @plfiorini can give us a feedback about the latest changes before merging would be great.

**Comment by plfiorini** [comment]
lgtm!

**Comment by emersion** [comment]
Thanks for the reviews!

A `.lJ9C.vtt.swp` has been committed, can you remove it?

Also, can you squash these two commits into a single one?

File: assets/subtitles/lJ9C.vtt [hunk 4]
```diff
@@ -190,7 +190,7 @@ from important i3 programs
 
 48
 00:02:10.000 --> 00:02:11.000
-like swaybar,
+like sway-bar,
```
Comment (emersion) [inline assets/subtitles/lJ9C.vtt:193]: Actually, the name of the executable is `swaybar`.

Comment (lucrus73) [reply assets/subtitles/lJ9C.vtt:193]: The screencast showing "man sway-bar" mislead me to think that was the name of the executable. I'm going to fix it in the next commit, thanks.

**Pull request merged**